### ISSUE TITLE
Apply animation button executes only once

### ISSUE THREAD

**Issue by lea10up** [10up/component-animate#9]
When you select your first set of library options, you can only apply the animation once on initial button press.

"Apply this Animation" only executes again when new library options are selected.

**Comment by timwright12** [comment]
TO DO: Clear the animation timer in the demo

**Comment by nicholasio** [comment]
Closing in favor of https://github.com/10up/component-library/issues/25
